kv/dist_sender.go

```diff
@@ -467,13 +468,10 @@ func (ds *DistSender) sendRPC(trace *tracer.Trace, raftID proto.RaftID, replicas
 
 	replies, err := ds.rpcSend(rpcOpts, "Node."+args.Method().String(),
 		addrs, getArgs, getReply, ds.gossip.RPCContext)
-	if err == nil {
-		// Set content of replies[0] back to reply
-		dst := reflect.ValueOf(reply).Elem()
-		dst.Set(reflect.ValueOf(replies[0]).Elem())
+	if err != nil {
+		return nil, err
 	}
-
-	return err
+	return replies[0].(proto.Response), nil
 }
```

Comment:
the reflection stuff? it's still here, just moved to `Send`. It's basically doing `*reply = newReply`. Hurray generics.